.github/workflows/_win-test.yml

```diff
@@ -122,6 +122,11 @@ jobs:
         run: |
           tree /F C:\$Env:GITHUB_RUN_ID\build-results
 
+      - name: Install test dependencies
+        shell: bash
+        run: |
+          ${CONDA_RUN} python -mpip install -r .github/requirements/pip-requirements-${{ runner.os }}.txt
```

Comment:
I add this for the Windows CI because the dependency in `setup.py` is not installed correctly.

Reply:
cc @malfet

Reply:
cc: @ZainRizvi would it be possible to move Windows CI a few steps closer to other OSes by using wheel installation?

Reply:
Sounds reasonable, not sure why we don't do that already. 

cc @huydhn in case you know why we don't know wheels for Windows CI today

Reply:
I'm not sure about the historical context, but we didn't build wheel for Windows in the past.  We just installed PyTorch instead `python setup.py install --cmake`, thus the use of 7z-ing.  The good news is that this has already been change by https://github.com/pytorch/pytorch/commit/3cdf621fe5c8f8378fda209b8a143443d33b2086#diff-4e1dd3a0537c1f0f7bc80a030a72c818c4a1aae8c5c2fd62752e1c6b1d1d6288, in which the wheel is now built by `python setup.py bdist_wheel`.  The 7z part remains because changing that is not the intention of the PR.  Let me build on that and submit a change to use the built wheel instead

Reply:
Removed since #94958 is landed.

- #94958